.github/workflows/release-docker-image-server-only.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release UID2 Examples Image for Server Only
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} Server Only Docker Image by @${{ github.actor }}  
```

Comment:
This should be either Publish Pre-release UID2 Examples Image for Server Only or Release Major UID2 Examples Image for Server Only